ceph.spec.in

```diff
@@ -33,6 +30,9 @@ Source0:	http://ceph.com/download/%{name}-%{version}.tar.bz2
 %if 0%{?fedora} || 0%{?rhel}
 Patch0:		init-ceph.in-fedora.patch
 %endif
+#################################################################################
+# distro-unconditional dependencies
```

Comment:
"distro-independent dependencies" . . . dunno, sounds rather confusing. 

Originally I had written "dependencies that apply across all distros" and "dependencies that are specific only to a particular distro" but that seemed too verbose. Another option would be "non-distro-specific dependencies" and "distro-specific dependencies".